3-semantic-html/Activity.html

```diff
@@ -22,10 +22,10 @@ <h3>Semantic caracteristics:</h3>
     <main>
         <article>
              <header>
-                <h1>HTML</h1>                
+                <h2>HTML</h2>                
```

Comment:
This line was fine. I put the comment because it's better to have only one `<h1>` tag per page.
```suggestion
                <h1>HTML</h1>                
```